05-pages/content-types/Article.stories.js

```diff
@@ -0,0 +1,109 @@
+            { content: 'This is the first item in the unordered list.' },
+            {
+              label: 'This is the optional label',
+              content: 'And here is the item that goes with the label.',
+            },
+            { content: "Here's the third item." },
+            { content: "And here's the last item." },
+          ]}
+        />
+      </>
+    }
+  >
+    <Heading level={2}> Knowles Cognitive Information Processing reaction label draft training</Heading>
+    <ResponsiveImage
+      outputImage={true}
+      modifiers={['mb-space']}
+      image={{
+        alt: 'Castle',
+        src: 'https://source.unsplash.com/A_InfAQM_lU/1600x900',
+      }}
+    />
+    <Paragraph>
+      At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, At accusam aliquyam diam diam dolore dolores duo eirmod eos erat, et nonumy sed tempor et et invidunt justo labore Stet clita ea et gubergren, kasd magna no rebum. sanctus sea sed takimata ut vero voluptua. est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat.
+    </Paragraph>
+    <Paragraph>
+      Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, At accusam aliquyam diam diam dolore dolores duo eirmod eos erat, et nonumy sed tempor et et invidunt justo labore Stet clita ea et gubergren, kasd magna no rebum. sanctus sea sed takimata ut vero voluptua. est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat. 
+    </Paragraph>
+    <Hr />
+    <Grid gridLabel="Development label repeat mints select pedagogy whiteboard click next asynchronous learning record instructional design" gridType="card">
+      <Card
+        additionalClasses={['grid__item']}
+        outputImage={true}
+        image={{
+          alt: 'People',
+          src: 'https://picsum.photos/id/987/480/300',
+        }}
+        heading="Meet and Greet with the Chancellor"
+        subheading="Mark Namerson Visits the Students"
+        linkText="Continue Reading"
+        linkUrl="#"
+        children="Curabitur non nulla sit amet nisl tempus convallis quis ac lectus. Vivamus magna justo, lacinia eget consectetur sed, convallis at tellus. Donec rutrum congue leo eget malesuada. Proin eget tortor risus. Sed porttitor lectus nibh. Nulla quis lorem ut libero malesuada feugiat."
```

Comment:
While this gets the job done, children is a reserved property that is accessed by passing JSX to the component (which in this case is just a string.) Instead of
```
<Card
  prop1={...}
  prop2={...}
  children="text here"
/>
```
Try:
```
<Card
  prop1={...}
  prop2={...}
>
  text here
</Card>
```

Reply:
@nJim I updated the `card` component to address your comment here. Could you please give this another look?

Thank you!